src/quo2/components/list_items/dropdown.cljs

```diff
@@ -0,0 +1,148 @@
+(ns quo2.components.list-items.dropdown
+  (:require [quo.react-native :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.reanimated :as reanimated]
+            [reagent.core :as reagent]))
+
+(defn apply-anim [dd-height val]
+  (reanimated/animate-shared-value-with-delay dd-height
+                                              val
+                                              300
+                                              :easing1
+                                              0))
+
+(def sizes {:big    {:icon-size 20
+                     :font      {:font-size :paragraph-1}
+                     :height    40
+                     :padding   {:padding-with-icon    {:padding-vertical   9
+                                                        :padding-horizontal 12}
+                                 :padding-with-no-icon {:padding-vertical 9
+                                                        :padding-left     12
+                                                        :padding-right    8}}}
+            :medium {:icon-size 20
+                     :font      {:font-size :paragraph-1}
+                     :height    32
+                     :padding   {:padding-with-icon    {:padding-vertical   5
+                                                        :padding-horizontal 8}
+                                 :padding-with-no-icon {:padding-vertical 5
+                                                        :padding-left     12
+                                                        :padding-right    8}}}
+            :small  {:icon-size 12
+                     :font      {:font-size :label}
```

Comment:
I think the sizes for small are incorrect. 
As far as I can see, small uses 13/Medium (paragraph-2)

and the icons seem too small too. 🤔 
<img width="579" alt="Screenshot 2022-09-20 at 12 20 51" src="https://user-images.githubusercontent.com/22799766/191245649-8f0cc38e-8d24-4bf1-86e1-cd1489472957.png">

The black text/icons are the implementation and the white is the figma

Reply:
Fixed, Thank you so much Jamie for taking such a close look
![image](https://user-images.githubusercontent.com/33176106/191288069-c32f80e5-1da6-4620-88fd-d20e3d9c73ed.png)